File: count prime numbers.cpp
#include <iostream>
#include <cmath>

bool is_prime(int n) {
    if (n <= 1) return false;
    for (int i = 2; i <= std::sqrt(n); ++i) {
        if (n % i == 0) return false;
    }
    return true;
}

int main() {
    int N;
    std::cout << "Enter N: ";
    std::cin >> N;

    int count = 0;
    for (int i = 2; i <= N; ++i) {
        if (is_prime(i)) count++;
    }

    std::cout << "Prime count: " << count << std::endl;
    return 0;
}

File: Loop/Count_Prime_Numbers.py
number=int(input("Enter the number: "))
count=0
for num  in range (2, number+1):
        isPrime=1
        for check in range(2, num):
            if num%check==0:
                isPrime=0
        
        if isPrime==1:
            print("The number is:",num)
            count+=1
print("The total count of Prime Number is: ", count)





        
            
    
 
        





